src/Microsoft.AspNet.OData.Shared/CompatibilityOptions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+using System;
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Contains bitmasks for features that need backward compatibilty.
+    /// </summary>
+    [Flags]
+    public enum CompatibilityOptions
```

Comment:
>CompatibilityOptions [](start = 16, length = 20)

Compatibility naming sounds like to compatibility to a certain previous version, Right? But, The main of this options is for the propose?